decidim/decidim
# Fix for liking a resource prompts for groups even though groups are disabled

#### :tophat: What? Why?
While working on #13373 I have encountered some specs that failed. Upon investigation, i noticed that the select identities pop-up is being fired regardless the organization setting of user_group_enable value. 
Additionally, i have aligned the 3 endorse spec files that i have found to ensure a consistent testing. 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to
- Fixes  #13383

#### Testing
1. login as administrator and visit a proposal 
2. press like in a resource and notice the `select identity` pop-up is displayed 
3. Visit admin and uncheck "user group enabled", then save
4. repeat step 2, and see the `select identity` is still being shown 
5. Apply patch 
6. See the elect identity is not being displayed, the like is provided. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
